### ISSUE TITLE
Unable to checkout recent git master because of too long file name

### ISSUE THREAD

**Issue by kaltsi** [openSUSE/open-build-service#1872]
This commit d733d06149620ad682efe5dd8a29b78a3ee63aa1 introduces a file with 165 char filename to the git repository.

It's not possible to check out the current git master if you try to do it on an ecryptfs encrypted filesystem. You'll get an error like this when cloning:

```
error: unable to create file src/backend/testdata/buildinfo/srcserver/getprojpack?withsrcmd5&withdeps&withrepos&expandedrepos&withremotemap&ignoredisable&project=openSUSE:13.2&repository=standard&arch=i586&parseremote=1&package=screen (File name too long)
```

Maybe there's nothing you want to do about it, just letting you guys know that that file is problematic in a case like this.

**Comment by hennevogel** [comment]
@M0ses please decide

**Comment by kaltsi** [comment]
One use case for ecryptfs is when a user chooses not to use full disk encryption on their computer, but chooses to encrypt only homedir.

**Comment by M0ses** [comment]
Another file has to be shortend:

'src/backend/testdata/buildinfo/srcserver/getprojpack?withsrcmd5&withdeps&withrepos&withconfig&withremotemap&ignoredisable&project=openSUSE:13.2&repository=standard&arch=i586&package=screen'

**Comment by kulmesa** [comment]
there is still one file in master causing problems:

`$ git clone https://github.com/openSUSE/open-build-service.git`
...
`error: unable to create file src/backend/testdata/buildinfo/srcserve /getprojpack?withsrcmd5&withdeps&withrepos&withconfig&withremotemap&ignoredisable&project=openSUSE:13.2&repository=standard&arch=i586&package=screen (File name too long)`
`fatal: unable to checkout working tree`

**Comment by M0ses** [comment]
@kulmesa - Problem still there ? If so - please feel free to reopen this issue

**Comment by kaltsi** [comment]
@M0ses we don't have the power to reopen closed issues, but the problem still exists, as @kulmesa reported.

**Comment by M0ses** [comment]
Sorry, cannot reproduce:


```
# git clone https://github.com/openSUSE/open-build-service.git
....
# cd open-build-service/
# ls src/backend/testdata/buildinfo/srcserver
/bin/ls: cannot access 'src/backend/testdata/buildinfo/srcserver': No such file or directory
```

**Comment by kulmesa** [comment]
seems to work now. I guess it was fixed in commit: 5b8535aef6cb9c232bee26b25bfdf20e2d0bb6ba

**Comment by M0ses** [comment]
Great - Thx for feedback
